src/mudlet.cpp

```diff
@@ -1194,6 +1194,31 @@ void mudlet::commitLayoutUpdates()
     }
 }
 
+bool mudlet::setUserWindowFontSize(Host* pHost, const QString& name, int size)
+{
+    if (!pHost) {
+        return false;
+    }
+
+    QMap<QString, TConsole*>& dockWindowConsoleMap = mHostConsoleMap[pHost];
+    QMap<QString, TDockWidget*>& dockWindowMap = mHostDockConsoleMap[pHost];
+
+    if (dockWindowMap.contains(name) && dockWindowConsoleMap.contains(name)) {
+        qDebug() << "Setting UserWindow FontSize to" << size << " for: " << name;
+        TConsole* pC = dockWindowConsoleMap[name];
+        pC->console->mDisplayFont = QFont("Bitstream Vera Sans Mono", size, QFont::Normal);
+        pC->console->updateScreenView();
```

Comment:
Could you add a "why" comment here as to why we need to do it twice - just so we remember in the future why